packages/dashmate/docker/entrypoint.sh

```diff
@@ -6,25 +6,26 @@ USERNAME=dashmate
 GROUP=docker
 
 # check if user with our uid exists in the system
-if [ ! $(getent passwd $USER_ID | grep $USER_ID -q) ]; then
+if ! (getent passwd $USER_ID | grep -q $USER_ID); then
   echo "Creating user $USERNAME ($USER_ID)"
```

Comment:
Why is username is important? Can we use own username with same ID inside docker container and just rewrite DASHMATE_HOME_DIR for helper. So we don't need to deal with symlinks.

Reply:
In Ubuntu you will get this - `adduser: uid '1000' in use`

Reply:
Why do we create a user with an existing ID?